apidoc/validate.py

```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I'd suggest we simply fix the validate.js file to include the additional checks. I don't think it's good for us to have two separate and unequal validation scripts in entirely different language/toolsets.